server/plugin/webhook.go

```diff
@@ -459,7 +459,13 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 func (p *Plugin) sanitizeDescription(description string) string {
 	var policy = bluemonday.StrictPolicy()
 	policy.SkipElementsContent("details")
-	return strings.TrimSpace(policy.Sanitize(description))
+	// Replacing the unicodes with the respective special characters to have proper rendering in code block.
+	result := strings.ReplaceAll(policy.Sanitize(description), "&#39;", "'")
+	result = strings.ReplaceAll(result, "&#34;", "\"")
+	result = strings.ReplaceAll(result, "&amp;", "&")
+	result = strings.ReplaceAll(result, "&gt;", ">")
+	result = strings.ReplaceAll(result, "&lt;", "<")
+	return strings.TrimSpace(result)
```

Comment:
I wonder if there is a more catch-all holistic approach

Reply:
@mickmister I explored other packages to sanitize the HTML but the one we are using right now was the best I could find. But this converts the special characters to their unicode value which are not formatted inside a code block, so we are replacing them again here. Please feel free to share your opinions if you have a better approach in mind.

Reply:
Hmm so it's the `policy.Sanitize` that's causing the issue. We really just want to use it to remove the `<details>` elements from the description, and not escape those characters.

As this comment https://github.com/microcosm-cc/bluemonday/issues/39#issuecomment-454358002 mentions on a related issue for the `bluemonday` package, we can call `html.UnescapeString` from the result to make those characters go back to what we want them to be

Reply:
@mickmister Updated